### PR TITLE
Reverts Adds Neon Carpet (#55782)

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

It appears to have been causing massive amounts of maptick, this'll need a testmerge so we can test my hypothesis 

## Why It's Good For The Game

We're all trapped in the belly of this horrible machine, and the machine is bleeding to death

### CODE REVIEW

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -419,11 +419,14 @@
...
-	if((last_update_overlay == update_overlay) && (update_state & UPSTATE_ALLGOOD))
-		. |= COMSIG_ATOM_NO_UPDATE_OVERLAYS
+	var/results = 0
+	if(last_update_state == update_state && last_update_overlay == update_overlay)
+		return 0
+	if(last_update_state != update_state)
+		results += 1
+	if(last_update_overlay != update_overlay)
+		results += 2
+	return results
```
Comment (LemonInTheDark) [inline code/modules/power/apc.dm:429]: I cannot verbally describe how much it hurts to readd this

**Comment by Pugie** [comment]
Sounds intended and a feature to me

File: code/modules/power/apc.dm (hunk 1 above)
Comment (TemporalOroboros) [reply code/modules/power/apc.dm:429]: APCs really need a refactor at some point.

**Comment by Timberpoes** [comment]
As Potato has mentioned, this needs TMing during highpop.

**Comment by TemporalOroboros** [comment]
I'm honestly surprised they got merged without a TM in the first place.

**Comment by Rohesie** [comment]
@actioninja warned us about this

**Comment by Timberpoes** [comment]
image

**Pull request merged**